File: 8 kyu/Return Negative/Kata.kt
class Kata {

    fun makeNegative(x: Int): Int {
        if (x > 0) return -x
        else return x
    }
}

class Kata {

    fun makeNegative(x: Int) = if(x <= 0) x else -x
}

import kotlin.math.abs

class Kata {
    fun makeNegative(x: Int) = -abs(x)
}

File: 8 kyu/Return Negative/Kata.cs
using System;

public static class Kata
{
    public static int MakeNegative(int number)
    {
        return number > 0 ? -number : number;
    }
}


public static class Kata
{
    public static int MakeNegative(int number)
    {
        return -Math.Abs(number);
    }
}